modules/certificate-authority-service/outputs.tf

```diff
@@ -17,25 +17,25 @@
 output "ca_ids" {
   description = "The CA ids."
   value = {
-    for k, v in google_privateca_certificate_authority.cas
+    for k, v in google_privateca_certificate_authority.default
     : k => v.id
   }
 }
 
 output "ca_pool" {
   description = "The CA pool."
-  value       = try(google_privateca_ca_pool.ca_pool[0], null)
+  value       = try(google_privateca_ca_pool.default[0], null)
```

Comment:
Can you remove the try? I think either `one(google_privateca_ca_pool.ca_pool)` should work

Reply:
I ran into issues with `destroy` in the past without a `try` in this kind of situation, but let's change it and if it breaks we revert.